From 81f46f453be0f5f7b331fb624cc0b04247daae65 Mon Sep 17 00:00:00 2001
From: Manikandan Somasundaram <manikandan@meta.com>
Date: Mon, 11 Sep 2023 21:33:38 -0700
Subject: [PATCH] Fix oss build breakage.

Summary: This fixes the build breakage caused by D48619419

Reviewed By: praihan

Differential Revision: D49172793

fbshipit-source-id: 682200661feab78cd22ea6981020fc220681a670
---
 thrift/lib/cpp/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/thrift/lib/cpp/CMakeLists.txt b/thrift/lib/cpp/CMakeLists.txt
index 16ca5da2fb2..5363752ed82 100644
--- a/thrift/lib/cpp/CMakeLists.txt
+++ b/thrift/lib/cpp/CMakeLists.txt
@@ -92,6 +92,7 @@ add_library(
   async/THttpAsyncChannel.cpp
   async/TUnframedAsyncChannel.cpp
   server/TServerObserver.cpp
+  ../cpp2/detail/EventHandlerRuntime.cpp
 )
 if (
   CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR
